x-pack/plugins/maps/public/classes/styles/heatmap/components/__snapshots__/heatmap_style_editor.test.tsx.snap

```diff
@@ -48,10 +48,10 @@ exports[`HeatmapStyleEditor is rendered 1`] = `
           "getPalette": [Function],
           "palette": Array [
             "#e0e4eb",
-            "#c2c9d5",
-            "#a6afbf",
-            "#8c95a5",
-            "#757c8b",
+            "#c3c9d4",
```

Comment:
I could not find any changes in the logs that resulting in color picker snapshot changes. Can you explain why these hex codes changed? The are generated with `euiPaletteGray`.

Reply:
Hey Nathan! I was a little confused about this snapshot change too, but it looks like the change likely occurred due to our underlying `chroma.js` dependency upgrade: https://github.com/elastic/eui/pull/7039

@tkajtoch, do you mind jumping in here if you would consider these changes expected? FWIW, I'm looking at our docs' palettes [before](https://eui.elastic.co/v86.0.0/#/utilities/color-palettes#recommended-quantitative-palettes) and [after](https://eui.elastic.co/v87.1.0/#/utilities/color-palettes#recommended-quantitative-palettes) and I can barely detect a difference to the human eye, so it's likely not a major impact.

Reply:
Also, just wanted to mention our general changelog philosophy - we've historically skipped logging tech debt items / only log items that would directly impact consumer usage. We didn't expect this one to impact consumers or end-users, but the wonderful Javascript dependency ecosystem being what it is, an unexpected side effect emerged 😬 (although it's fortunately fairly benign in this case).

From a general developer experience perspective, I'd be curious to hear if you think it's worth logging all changes, including dependency upgrades, and whether the extra noise would be worth the extra traceability!

Reply:
I looked into the changes and compared updated colors. Usually, tools like chroma.js convert input colors to a broader color space before calculating color variations, specifically to have a long enough bit depth for the colors to look correct after processing. 64-bit float values in HSL color space are way more precise than 8-bit RGB. I'm 99% sure it's caused by switching to using a [binomial of degree n in this commit](https://github.com/gka/chroma.js/commit/2a543d14645e0c706b10c5fbdc6af16f04be4d5f#diff-2b165938e307af7902dca406408325f88951be2175e65a4ca531ed9a58af3734). We see the changes in `chroma.scale` because it's used [here](https://github.com/gka/chroma.js/blob/main/src/generator/bezier.js#L67) and probably in other places as well that eventually affect `chroma.scale` results.

The color difference can't realistically be noticed and even though hue and saturation are updated for each color it just looks like some kind of gamma correction while keeping luminance the same.

Reply:
Thanks for the explanation.

> From a general developer experience perspective, I'd be curious to hear if you think it's worth logging all changes, including dependency upgrades, and whether the extra noise would be worth the extra traceability!

I think its worth putting all changes in the change log. So there is a single comprehensive list of differences to consult.

Reply:
Thanks Nathan! I'll take that feedback back to the team and discuss. No promises on timelines, but agreed that it's bitten us in the past before and will potentially make a difference in the future for debugging.